independent-publisher-connectors/DPIRD Weather/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3611 @@
+              "title": ""
+            },
+            "summaries": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/stationSummaryModel"
+              }
+            }
+          }
+        }
+      }
+    },
+    "stationSummaryModel": {
+      "type": "object",
+      "properties": {
+        "period": {
+          "type": "object",
+          "properties": {
+            "from": {
+              "type": "string",
+              "title": "Summary Interval Start",
+              "description": "The start of the specified interval in UTC"
+            },
+            "to": {
+              "type": "string",
+              "title": "Summary Interval End",
+              "description": "The end of the specified interval in UTC."
+            },
+            "year": {
+              "type": "integer",
+              "format": "int32",
+              "title": "Summary Year",
+              "description": "This is the year value that the summary relates to."
+            },
+            "month": {
+              "type": "integer",
+              "format": "int32",
+              "title": "Summary Month",
+              "description": "This is the month value that the summary relates to."
+            },
+            "day": {
+              "type": "integer",
+              "format": "int32",
+              "title": "Summary Day",
+              "description": "This is the day of the month that the summary relates to."
+            },
+            "hour": {
+              "type": "integer",
+              "format": "int32",
+              "title": "Summary Hour",
+              "description": "This is the 24-hour value that the summary relates to."
+            },
+            "minute": {
+              "type": "integer",
+              "format": "int32",
+              "title": "Summary Minute",
+              "description": "This is the minute value that the summary data relates to. In case of a 15 minute or 30 minute interval, the minute value will represent the end of the interval."
+            }
+          },
+          "description": "The time-period of the specified interval."
+        },
+        "airTemperature": {
+          "type": "object",
+          "properties": {
+            "min": {
+              "type": "number",
+              "format": "float",
+              "title": "AirTempMin",
+              "description": "The minimum value observed within the period. In case of a daily summary (interval = 'daily'), this value is the minimum value observed in the 24 hours to 9 am on the specified day."
+            },
+            "minTime": {
+              "type": "string",
+              "title": "AirTempMinTime",
+              "description": "The date and time the minimum value was observed within the period in UTC. In case of a daily summary (interval = 'daily'), this value is date and time the mimimum value is observed in the 24 hours to 9 am on the specified day."

```

Comment:
It seems that not all grammar errors have been resolved, please review them. We advise to use some grammar/spelling checking tool.

Reply:
this is cut/paste from source API. Using code spell checker nothing shows up here. Which grammar tool are you using? If I use the same it will minimise this issue